indico/web/assets/bundles.py

```diff
@@ -14,6 +14,8 @@
 # You should have received a copy of the GNU General Public License
 # along with Indico; if not, see <http://www.gnu.org/licenses/>.
 
+from __future__ import unicode_literals
```

Comment:
Usually not a good idea to add it to existing files - it probably works here but there are other (legacy) files where this  would break things (I actually removed this from the isort config since it's more annoying than useful for already-existing files)